### PR TITLE
un-schedule access_beta_distribution for sle-micro

### PR DESCRIPTION
**Pull request by JRivrain**
Added commented schedule to default, so we just need to [un]comment the default file when needed.

### CODE REVIEW

**Comment by tinawang123** [comment]
LGTM

File: schedule/yast/sle/flows/default_slem.yaml [hunk 1]
```diff
@@ -4,7 +4,8 @@ bootloader:
   - installation/bootloader_start
 setup_libyui:
   - installation/setup_libyui
-access_beta_distribution: []
+# access_beta_distribution:
```
Comment (lemon-suse) [inline schedule/yast/sle/flows/default_slem.yaml:7]: Why not removed and just comments here?

**Comment by lemon-suse** [comment]
Finger in cross. And please remember to create a new PR to fix the comments.

File: schedule/yast/sle/flows/default_slem.yaml (hunk 1 above)
Comment (JRivrain) [reply schedule/yast/sle/flows/default_slem.yaml:7]: so we just need to un-comment when when we have beta again

**Pull request merged**